App.jsx

```diff
@@ -37,8 +38,22 @@ const App = () => {
             headerTintColor: '#CEC269',
             headerTitleStyle: {
               fontWeight: 'bold',
-
-            }
+            },
+          })}
+        />
+        <Stack.Screen
+          name='view by category'
+          component={ViewByCategory}
+          options={() => ({
+            headerBackTitle: 'Back',
+            title: 'Back',
+            headerStyle: {
+              backgroundColor: '#111518',
+            },
+            headerTintColor: '#CEC269',
+            headerTitleStyle: {
+              fontWeight: 'bold',
+            },
```

Comment:
We can refactor all the styling into a `screenOptions` as an attribute of the `Stack.Navigator`. 🚀

Reply:
👍🏼